tests/WithMocks.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace Tests;
+
+use Mockery;
+use Carbon\Carbon;
+use DoSomething\Gateway\Northstar;
+use Illuminate\Support\Facades\Storage;
+use DoSomething\Gateway\Resources\NorthstarUser;
+
+trait WithMocks
+{
+    /**
+     * The Faker instance for the request.
+     *
+     * @var \Faker\Generator
+     */
+    protected $faker;
+
+    /**
+     * Configure mocks for the application.
+     */
+    public function configureMocks()
+    {
+        // Reset mocked time, if set.
+        Carbon::setTestNow(null);
+
+        // Fake the storage driver.
+        Storage::fake('public');
+
+        // Get a new Faker generator from Laravel.
+        $this->faker = app(\Faker\Generator::class);
+        $this->faker->addProvider(new \FakerNorthstarId($this->faker));
```

Comment:
hi @DFurnes ! I was using Rogue as an example to set up mocks for tests. When I run this first test however, I get 
```
There was 1 error:

1) Tests\Unit\CallPowerTest::testFailedPostWithInvalidCallPowerCampaignId
Error: Class 'FakerNorthstarId' not found

/home/vagrant/DoSomething.org/chompy/tests/WithMocks.php:33
/home/vagrant/DoSomething.org/chompy/tests/TestCase.php:28
```
and I'm not sure why? Is there anything I have to do that I'm missing so it can register this line? I tried to put it in a `use` statement at the top of the file but that didn't work either...

Reply:
@chloealee It's saying that the class isn't found because that's a file from Rogue! Each application is its own self-contained "package", and can't directly reference code from other codebases (that's why we expose APIs).

Here's [the file that's referencing](https://github.com/DoSomething/rogue/blob/master/database/faker/FakerNorthstarId.php)  in Rogue, which you'd need to copy over to use here.

Reply:
thanks, @DFurnes ! I thought I did that with [this file](database/faker/FakerNorthstarId.php) though in this PR?

Reply:
Oh I just needed to do a `composer update`!

Reply:
Ahhh sorry! Was looking too fast! 🤦‍♂️